js/apps/admin-ui/src/clients/authorization/SearchDropdown.tsx

```diff
@@ -33,13 +33,17 @@ type SearchDropdownProps = {
   search: SearchForm;
   onSearch: (form: SearchForm) => void;
   isResource?: boolean;
+  isPolicies?: boolean;
+  isPermissions?: boolean;
```

Comment:
Rather than having a bunch of separate booleans, perhaps it would make sense to have a single 'type' field? This reduces the complexity of conditions in this component. 
```suggestion
  type:  'resource' | 'policy' | 'permission';
```

Reply:
yeah, sounds good to me!